test/test_manifest.json

```diff
@@ -952,6 +952,25 @@
        "enableXfa": true,
        "type": "eq"
     },
+    {  "id": "xfa_bug1720182",
+       "file": "pdfs/xfa_bug1720182.pdf",
+       "md5": "1351f816f0509fe750ca61ef2bd40872",
+       "link": true,
+       "rounds": 1,
+       "enableXfa": true,
+       "type": "eq",
```

Comment:
Given that the bug is about printing, shouldn't this also use `"print": true,` as well?